torch/nn/modules/module.py

```diff
@@ -419,8 +424,16 @@ def to(self, *args, **kwargs):
                 raise TypeError('nn.Module.to only accepts floating point '
                                 'dtypes, but got desired dtype={}'.format(dtype))
 
+        # memory_format handled separately as we can't call `to` operator with
```

Comment:
I don't really understand the comment.  Why can't you call to with only memory format?  Is it because it's not bindable yet? (list an issue here).  Or it's not part of the API (describe why?)

Reply:
Fixed.